test/cpp/interop/xds_interop_client.cc

```diff
@@ -154,11 +158,20 @@ class TestClient {
             std::chrono::system_clock::now() + std::chrono::seconds(INT_MAX);
       }
     }
+    SimpleRequest request;
+    if (absl::GetFlag(FLAGS_response_payload_size) > 0) {
+      request.set_response_size(absl::GetFlag(FLAGS_response_payload_size));
+    }
+    if (absl::GetFlag(FLAGS_request_payload_size) > 0) {
+      std::string payload(absl::GetFlag(FLAGS_request_payload_size), '\0');
```

Comment:
Wait, is this a `0` char? I thought `\0` is the null char, e.g. `0x0`.

Reply:
I left it as it for now. I looked at Java's implementation https://github.com/grpc/grpc-java/blob/master/interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java#L501 and it looks like it's the null character as well?

Reply:
There are quite some inconsistencies among the existing interop client / server (not this Xds client / server) how we use the null char. But that doesn't seem to matter much at all, because only the length is being checked.

Reply:
Chatted offline with @yashykt, setting this to the zero `0` character now.